builder/src/js/lexer.js

```diff
@@ -1,14 +1,11 @@
-import tokenizer from "./tokenizer";
-
-//의미를 분석. value랑 type을 분류해서 객체형태로 저장해줌.
 const lexer = (tokenizerArr) => {
     let result = [];
-    let type = ""
     tokenizerArr.map(function(token){
```

Comment:
P2
map 함수를 사용하기 적합한 코드이긴 한데 올바르게 사용되지 않은 것 같아요. 아래와 같이 각 요소들에 어떤 함수를 적용한 새로운 배열을 반환하는 형태로 사용되어야 합니다.
```suggestion
return tokenizerArr.map(token => { });
```